### PR TITLE
React setup (PR)

### PR DESCRIPTION
**Pull request by teke85**
### The requirements of this project were as follows:-

- Set up a React app.
- Use React documentation.

# General Requirements

- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config). Remember that this is your first React project and it requires a special linters setup. Check the instructions in the linters-config README.

- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).

- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

# Outcome

React was Setup successfully as per project requirement
React-App

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+- To add 3 pages to the website which will display different information
+
+# Getting Started
+
+**To get a local copy up and running follow these simple steps.**
+
+Run the command below from the terminal
+
+- git clone in your terminal
```
Comment (OyePriscilla) [inline README.md]: - [ ] Kindly add the link to your [repo](git@github.com:Tekhlay/math-magicians.git) so as to make it available to all that may love to access your work. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @teke85 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Linters check passed ✔️ 
Git flow observed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (teke85) [reply README.md]: Hi @OyePriscilla thank you for your code review. I have added the link to my repo
teke85-Math-Magician-at-React_Setup

**Review by OyePriscilla — APPROVED** [review]
Hi @teke85 👍 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters check passed ✔️ 
Git flow observed 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**